enterprise_knowledge_retriever/src/document_retrieval.py

```diff
@@ -144,39 +143,36 @@ def get_config_info(self):
         # Read config file
         with open(CONFIG_PATH, 'r') as yaml_file:
             config = yaml.safe_load(yaml_file)
-        api_info = config["api"]
-        llm_info =  config["llm"]
-        embedding_model_info = config["embedding_model"]
-        retrieval_info = config["retrieval"]
-        prompts = config["prompts"]
-        prod_mode = config["prod_mode"]
-        
-        return api_info, llm_info, embedding_model_info, retrieval_info, prompts,prod_mode
-    
+        api_info = config['api']
+        llm_info = config['llm']
+        embedding_model_info = config['embedding_model']
+        retrieval_info = config['retrieval']
+        prompts = config['prompts']
+        prod_mode = config['prod_mode']
+
+        return api_info, llm_info, embedding_model_info, retrieval_info, prompts, prod_mode
 
     def set_llm(self):
         if self.prod_mode:
```

Comment:
Is it better to move this part to utils/ since it will be used by all the kits? Also, will be easier to change in case we change the name of SAMBANOVA_API_KEY or API gateway in the future.

Reply:
this is also present in financial assistant and function calling, Should we handle this in a following PR?